prog/analysis.go

```diff
@@ -93,6 +95,8 @@ func (s *state) analyzeImpl(c *Call, resources bool) {
 						val = val[:len(val)-1]
 					}
 					s.files[val] = true
+				case BufferGlob:
+					s.globs[val] = true
```

Comment:
I don't see s.globs used anywhere. If it's not used, there is no point in collecting glob values during analysis.
Glob argument values are file names, right? So I think it should go into s.files. You mentioned some failed tests, but you did not say what tests are failing and how.

Reply:
The 2nd panic will be triggered if using s.files

```
func (r *randGen) filename(s *state, typ *BufferType) string {
	fn := r.filenameImpl(s)
	if fn != "" && fn[len(fn)-1] == 0 {
		panic(fmt.Sprintf("zero-terminated filename: %q", fn))
	}
	if escapingFilename(fn) {
		panic(fmt.Sprintf("sandbox escaping file name %q, s.files are %v", fn, s.files))
	}
```

Reply:
removed s.globs

Reply:
I see. I think it's fine to just not store them at all, the same happens with openat for fixed files I think.